spec/03-plugins/bot-detection/01-access_spec.lua

```diff
@@ -0,0 +1,150 @@
+local helpers = require "spec.helpers"
+
+local HELLOWORLD = "HelloWorld"             -- just a test value
+local FACEBOOK = "facebookexternalhit/1.1"  -- matches a known bot in `rules.lua` 
+
+describe("Plugin: bot-detection", function()
+
+  local client
+  
+  setup(function()
+    helpers.kill_all()
```

Comment:
`kill_all` is not appropriate in `setup()` anymore. All integration tests should use `kong_stop()` in their `teardown()` context.